Funcky.Async/Extensions/AsyncEnumerableExtensions/AdjacentGroupByAwait.cs

```diff
@@ -0,0 +1,194 @@
+using System.Collections.Immutable;
```

Comment:
Overloads with cancellation support are missing

Reply:
All overloads implemented in b797f245916578ca4fd02d909b3ccd25ae5a99fc